Bottom: b607c58551cbb855d17d1fa953fe40bbddac0d36
Top:    5e7a6c574e7e73cc518e9dcea5a44bb12ed9283d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-18 18:40:49 +0000

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
